financialaid/backend.py

```diff
@@ -0,0 +1,35 @@
+"""
+Backend helper functions for financialaid
+"""
+from financialaid.constants import COUNTRY_INCOME_THRESHOLDS, DEFAULT_INCOME_THRESHOLD
+from financialaid.models import FinancialAidStatus
+
+
+def determine_tier_program(program, income):
+    """
+    Determines and returns the TierProgram for a given income.
+    """
+    # To determine the tier for a user, find the set of every tier whose income threshold is
+    # less than or equal to the income of the user. The highest tier out of that set will
+    # be the tier assigned to the user.
+    tier_programs_set = program.tier_programs.filter(current=True, income_threshold__lte=income)
+    tier_program = tier_programs_set.order_by("-income_threshold")[0]
```

Comment:
It was a long line and we wanted to split it up logically, according to the steps described by the comment above the line.